source/test/F0.CodeAnalysis.Testing/Shared/Check.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Composition;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace F0.Testing.Shared
+{
+	internal static class Check
```

Comment:
Maybe `Verify` is a better term?

Reply:
I agree. However, we already have `public static class Verify`, used by all analyzer tests, eventually calling into these `Check` methods.
I currently can't think of a better name.

Since this class is not public, we could change it into _Type-Extensions_ within a file called `TypeExtensions.Verify.cs` and prefix all methods with the name `Verify` as an alternative.
```csharp
internal static class TypeExtensions
{
    internal static void VerifyAccessibility(this Type type);
    internal static void VerifyNonInheritable(this Type type);
    // ...
}
```

Reply:
Sounds good 👍

Reply:
Great! This now follows _Clean Code / Chapter 2 / Pick One Word per Concept_.

I will also rename `StringExtensions.cs` to `StringExtensions.Formatting.cs`, which currently has `internal static string Untabify(this string code)`, in order to _Use Intention-Revealing Names_.